soc/arm/st_stm32/stm32mp1/Kconfig.defconfig.series

```diff
@@ -47,4 +47,11 @@ config IPM_STM32_IPCC
 
 endif # IPM
 
+if I2C_STM32
```

Comment:
This does not match the commit header message

Reply:
commit message and subject updated